io_scene_godot/export_godot.py

```diff
@@ -93,19 +94,13 @@ def export_object(self, obj, parent_gd_node):
         bpy.context.view_layer.objects.active = obj
 
         # Figure out what function will perform the export of this object
-        if (obj.type in converters.BLENDER_TYPE_TO_EXPORTER and
-                obj in self.exporting_objects):
+        if obj in self.exporting_objects:
             exporter = converters.BLENDER_TYPE_TO_EXPORTER[obj.type]
```

Comment:
Looks like `BLENDER_TYPE_TO_EXPORTER` still can not cover all the object types. Maybe use `get` which is safer, instead of `[]`. And also keep the `Unknown object type` warning for debugging purpose

Reply:
Correct, reverted.